jmx-scraper/src/main/java/io/opentelemetry/contrib/jmxscraper/util/StringUtils.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.contrib.jmxscraper.util;
+
+import javax.annotation.Nullable;
+
+public final class StringUtils {
```

Comment:
Seems silly, but elsewhere we take extra care to ensure that utilities like this are in an "internal" package and have a javadoc that says that they are internal and may change or be deleted at any time....so that users are discouraged from using them directly. We could do that here too.